README.md

```diff
@@ -84,7 +84,7 @@ Examples
 A few global configurations (applied to all tests) can be specified by
 environment variables:
 
-- `TEST_DIRECTORY`: Local directory that contains the test definition YAML
```

Comment:
`YAML` is the name of the format/language. I think it was correct in these places

Reply:
Just want to be consistent with our use of `yml`, not to dispute the official name of the language. I can put both `YAML` and `YML` there.